pkg/cmd/grafana-cli/commands/commands.go

```diff
@@ -172,6 +172,11 @@ var adminCommands = []*cli.Command{
 				Usage:  "Migrates passwords from unsecured fields to secure_json_data field. Return ok unless there is an error. Safe to execute multiple times.",
 				Action: runDbCommand(datamigrations.EncryptDatasourcePasswords),
 			},
+			{
+				Name:   "migrate-secrets",
```

Comment:
Any thoughts on that? 🤔  It may worth using a more concrete name or even enable subcommands. 
We may eventually want to introduce related commands for:
- Rolling back secrets (move from envelope encryption to legacy encryption).
- Rotate secrets (re-encrypt them with new keys).
- ...

Reply:
I agree, we need a more specific name, but I do not have any good suggestion atm.

Reply:
`reencrypt`?

Reply:
Yeah, every time I wrote "migrate secrets", my alternative thought fell back to "reencrypt secrets", so I agree it could be a better alternative! Thanks!

Reply:
How will we call re-encrypting evelope encrypted secrets using legacy way of encryption? rollback secrets?

Reply:
My initial proposal (tho I'm not super convinced is):

- `admin secrets-migration reencrypt`, for reencrypting secrets (the code present on this PR)
- `admin secrets-migration rotate-keys`, for reencrypting DEKs
- `admin secrets-migration rollback`, for rolling back to legacy encryption 

What do you think? Any thoughts? 🤔